CMakeLists.txt

```diff
@@ -235,7 +242,12 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR})
 set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
 
-set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++0x -Wall -Werror -Wuninitialized -Wvla")
+set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++11  -Wall -Werror -Wuninitialized -Wvla")
```

Comment:
@BSolonenko can we try to use cmake instead of gcc flags ?

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e